src/app/publications/publications-edit.component.html

```diff
@@ -0,0 +1,39 @@
+<form [formGroup]="publicationForm" class="publications-edit-form">
+  <div *ngIf="errorMessage" class="publications-edit-error">{{errorMessage}}</div>
+  <div fxLayout="column">
+    <div fxFlex>
+      <mat-form-field class="publications-edit-form-field">
+        <input matInput id="publications-edit-title" placeholder="Title" formControlName="title" [required]="true">
+      </mat-form-field>
+    </div>
+    <div fxFlex>
+      <mat-form-field class="publications-edit-form-field">
+        <input matInput [matDatepicker]="publicationDatePicker" [max]="maxDate"  placeholder="Date" formControlName="date">
+        <mat-datepicker-toggle matSuffix [for]="publicationDatePicker"></mat-datepicker-toggle>
+        <mat-datepicker #publicationDatePicker></mat-datepicker>
+      </mat-form-field>
+    </div>
+    <div fxFlex>
+      <mat-form-field class="publications-edit-form-field">
+        <input matInput  placeholder="Publisher or location of confluence, etc." formControlName="publisher" [required]="true">
```

Comment:
There should be `conference` instead of `confluence`.

Reply:
thank you. done